Firestore/Source/Local/FSTLocalStore.mm

```diff
@@ -460,13 +460,17 @@ - (void)releaseQuery:(FSTQuery *)query {
     TargetId targetID = queryData.targetID;
 
     auto found = _targetIDs.find(targetID);
-    FSTQueryData *cachedQueryData = found != _targetIDs.end() ? found->second : nil;
-    if (cachedQueryData.snapshotVersion > queryData.snapshotVersion) {
-      // If we've been avoiding persisting the resumeToken (see shouldPersistQueryData for
-      // conditions and rationale) we need to persist the token now because there will no
-      // longer be an in-memory version to fall back on.
-      queryData = cachedQueryData;
-      _queryCache->UpdateTarget(queryData);
+
```

Comment:
Nit: the line above is logically grouped this section. Delete the newline?

Reply:
Done.